### PR TITLE
add version and channel properties to hab_install

### PR DESCRIPTION
**Pull request by mwrock**
Signed-off-by: Matt Wrock <matt@mattwrock.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,13 +42,21 @@ This resource is written as a Chef 12.5 custom resource.
 #### Properties
 
 * `install_url`: URL to the install script, default is from the [habitat repo](https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh)
+* `version`: The version of habitat to install (defaults to latest)
+* `channel`: The reease channel to install from (defaults to `stable`)
```
Comment (lamont-granquist) [inline README.md]: *release

File: resources/install.rb [hunk 2]
```diff
@@ -56,4 +58,11 @@ def hab_path
       "/bin/hab"
     end
   end
+
+  def hab_command
+    cmd = [ "bash #{Chef::Config[:file_cache_path]}/hab-install.sh" ]
+    cmd.push("-v #{new_resource.version}") if new_resource.version
+    cmd.push("-c #{new_resource.channel}") if new_resource.channel
+    cmd.join(" ")
```
Comment (lamont-granquist) [inline resources/install.rb]: the execute resource takes an array, although you'll have to bust up the -v and -c args into two elements

Comment (mwrock) [inline resources/install.rb]: It takes an array or string. This isn't pretty but seems better than spiting it up further.

Comment (lamont-granquist) [inline resources/install.rb]: ah you need to pull bash out of PATH as well.

yeah okay nm.

**Comment by eeyun** [comment]
LGTM Might be good to get a test in place for the channel feature though.

**Comment by mwrock** [comment]
I left out the channel test because the unstable channel being up is unreliable. However since the kitchen tests are just run adhoc anyways, I'll add this in the next round of updates.

**Pull request merged**